src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -89,7 +89,7 @@
 
 // FIXME: The transaction management here is getting out of hand. The annotations should probably
 // be removed entirely and replaced with explicit transaction management, and the protected methods
-// using them should be reverted to their propery visibility (private).
+// using them should be reverted to their property visibility (private).
```

Comment:
+1 on the spelling correction, but it was intended to be "proper" not "property"

Reply:
Oh haha yeah, that makes more sense!